azurerm/helpers/validate/strings.go

```diff
@@ -1,10 +1,29 @@
 package validate
 
 import (
+	"encoding/base64"
 	"fmt"
 	"strings"
 )
 
+// Base64EncodedString validates that the string is base64 encoded
+func Base64EncodedString(i interface{}, k string) ([]string, []error) {
```

Comment:
looks like we already have this function:
![image](https://user-images.githubusercontent.com/1638467/124615404-16f3c800-de2a-11eb-9990-013b22f89594.png)

lets maybe unify the two?

also could we place this in internal/tf/validate ?

Reply:
💯  have updated the other resource to use the common function